codalab/apps/web/static/js/worksheet/table_bundle_interface.js

```diff
@@ -32,11 +32,28 @@ var TableBundle = React.createClass({
             else
               this.focusOnRow(newIndex);
         }.bind(this), 'keydown');
+
+        Mousetrap.bind(['enter'], function(e) {
```

Comment:
The handler-setting code in the original location was actually being called regardless of whether the action bar was active or not. Decided to go this route because alternative was to pass `activeComponent` as an additional prop to each of the table rows.